src/coreComponents/constitutive/permeability/CarmanKozenyPermeability.hpp

```diff
@@ -0,0 +1,154 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 Total, S.A
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file CarmanKozenyPermeability.hpp
+ */
+
+#ifndef GEOSX_CONSTITUTIVE_PERMEABILITY_CARMANKOZENYPERMEABILITY_HPP_
+#define GEOSX_CONSTITUTIVE_PERMEABILITY_CARMANKOZENYPERMEABILITY_HPP_
+
+#include "constitutive/permeability/PermeabilityBase.hpp"
+
+
+namespace geosx
+{
+namespace constitutive
+{
+
+class CarmanKozenyPermeabilityUpdate : public PermeabilityBaseUpdate
```

Comment:
I propose to use KozenyCarman instead of CarmanKozeny to make it more consistent with literature.

Reply:
I think my Petrophysics professor always used CarmanKozeny and that 's why I am used to that but I am fine with changing it.